swift_package/BrainFlow/Sources/BrainFlowBindings/BrainFlowConstants.swift

```diff
@@ -0,0 +1,234 @@
+//
+//  BrainFlowConstants.swift
+//  a Swift binding for BrainFlow's enumeration types
+//
+//  Created by Scott Miller for Aeris Rising, LLC on 8/23/21.
+//
+
+import Darwin
```

Comment:
take a look please at all comments in a copypasted file from different folder and apply them here

Reply:
From which file?